packages/dids/src/methods/did-key.ts

```diff
@@ -345,83 +343,44 @@ export class DidKey extends DidMethod {
     keyManager?: TKms;
     options?: DidKeyCreateOptions<TKms>;
   } = {}): Promise<BearerDid> {
+    // Before processing the create operation, validate DID-method-specific requirements to prevent
+    // keys from being generated unnecessarily.
+
+    // Check 1: Validate that `algorithm` or `verificationMethods` options are not both given.
     if (options.algorithm && options.verificationMethods) {
       throw new Error(`The 'algorithm' and 'verificationMethods' options are mutually exclusive`);
     }
 
+    // Check 2: If `verificationMethods` is given, it must contain exactly one entry since DID JWK
```

Comment:
```suggestion
    // Check 2: If `verificationMethods` is given, it must contain exactly one entry since DID Key
```

Reply:
Fixed in 51e3a8f16ba3a34d8ab77f0ecb576a71b4e21a1a